source/Nevermore.Benchmarks/Model/Order.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Nevermore.Benchmarks.Model
+{
+    public class Order
+    {
+        public Order()
+        {
+
+        }
+
+        public Order(IEnumerable<(string, Type)> relatedDocuments)
+        {
+            RelatedDocuments = relatedDocuments?.ToArray();
+        }
+
+        public string Id { get; set; }
+        public string Name { get; set; }
+        public decimal Price { get; set; }
+
+        public IEnumerable<(string, Type)> RelatedDocuments { get; }
```

Comment:
The code I can find in OctopusServer has this property marked as `  [JsonIgnore]`. Should we do the same here?